### ISSUE TITLE
[BUG]: DLP Editor V2 app not showing all of the available HTTP connectors 

### ISSUE THREAD

**Issue by marcster** [microsoft/powerapps-tools#950]
**Describe the bug**
All of the HTTP connector(s) not showing in DLP Editor V2 app

**Component (please tell us which flow or app you are experiencing issues with):**
 - Solution Core
 - Version 2.0
 - DLP Editor V2

**To Reproduce**
1. Log into the DLP Editor V2 app
2. Create a tenant policy
3. On the connectors screen do a search for HTTP
4. Only the HTTP with Azure AD connector is seen.

**Expected behavior**
There should be three other HTTP connectors seen i.e. HTTP, HTTP with webhook and When a HTTP is request.

**Screenshots**
![image](https://user-images.githubusercontent.com/1025102/115562532-7cfe9380-a2ae-11eb-91db-706d04052f11.png)
![image](https://user-images.githubusercontent.com/1025102/115569814-7cb5c680-a2b5-11eb-93c2-1927d7c5ffb7.png)

**Additional context**

**Comment by JeneferM-MSFT** [comment]
This is a limitation in the tool that we will have to live with. These 3 things aren't treated as connectors in many ways, they are rather just built in to the product, hence class Built-In. 

Thank you for raising though so that we can have this noted for others.

**Comment by marcster** [comment]
Hi Jenefer. Thanks for the update about this.  Some additonal context around this. At the company I work for, we are on the Core version 1.89 due to the issue I raised around the Common Data Service connectors which was fixed (Thanks for this). So, now due to this HTTP issue, my manager is concerned about upgrading and using this app.  In the 1.89 version, we can see the HTTP connectors in the DLP Editor V2 version. So, does this mean this is now not possible anymore? This came to light because we implemented a DLP policy on our tenant which had the HTTP connector as blocked and we stopped one of our areas' functionality from working as it was not picked up in the impact analysis.  Appreciate all the hard work the team is doing on the toolkit.
